Plugins/Player/NWScript/nwnx_player.nss

```diff
@@ -427,6 +427,13 @@ object NWNX_Player_GetGameObject(object oPlayer);
 /// @param nMask A mask of OBJECT_UI_DISCOVERY_*, or -1 to clear the override
 void NWNX_Player_SetObjectUiDiscoveryMaskOverride(object oPlayer, object oObject, int nMask);
 
+/// @brief Send a party invite from oInviter to oPlayer
+/// @param oPlayer The player to invite
+/// @param oInviter The one inviting the player
+/// @param bForceInvite TRUE: Sends the invite even if the target ignores invites
```

Comment:
Not sure if worth noting, but there is also a clientside setting to ignore invites in general, and bForceInvite won't help with that.

Reply:
bSlightlyForceInvite